changelogs/fragments/1885-sanity-check-fixes-batch3.yml

```diff
@@ -0,0 +1,18 @@
+  - plugins/modules/cloud/oneandone/oneandone_firewall_policy.py - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
+  - plugins/modules/cloud/oneandone/oneandone_load_balancer.py - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
+  - plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
+  - plugins/modules/cloud/oneandone/oneandone_private_network.py - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
+  - plugins/modules/cloud/oneandone/oneandone_server.py - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
+  - plugins/modules/cloud/profitbricks/profitbricks.py - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
+  - plugins/modules/cloud/profitbricks/profitbricks_volume.py - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
+  - plugins/modules/cloud/webfaction/webfaction_domain.py - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
+  - plugins/modules/cloud/webfaction/webfaction_site.py - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
+  - plugins/modules/clustering/consul/consul.py - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
+  - plugins/modules/clustering/consul/consul_acl.py - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
+  - plugins/modules/clustering/consul/consul_session.py - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
+  - plugins/modules/monitoring/datadog/datadog_monitor.py - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
+  - plugins/modules/monitoring/sensu/sensu_check.py - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
+  - plugins/modules/monitoring/sensu/sensu_client.py - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
+  - plugins/modules/monitoring/sensu/sensu_handler.py - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
+  - plugins/modules/packaging/language/bundler.py - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
```

Comment:
```suggestion
  - oneandone_firewall_policy - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - oneandone_load_balancer - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - oneandone_monitoring_policy - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - oneandone_private_network - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - oneandone_server - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - profitbricks - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - profitbricks_volume - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - webfaction_domain - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - webfaction_site - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - consul - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - consul_acl - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - consul_session - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - datadog_monitor - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - sensu_check - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - sensu_client - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - sensu_handler - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - bundler - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
```

Reply:
Perfect eye, felixfontein:) I forgot to look at the fragment at all:)